tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -130,6 +152,17 @@ func TestMain(m *testing.M) {
 	}
 
 	log.Printf("Test log: %s\n", logFile)
+
+	// tmp dir setup
+	tmpDir, err := os.MkdirTemp(mntDir, "tmpDir")
+	if err != nil {
+		printAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+	}
+
+	// tmp file setup
```

Comment:
instead of just saying tmp file setup. Please mention why we are creating a temp file.